src/screens/Names.js

```diff
@@ -31,30 +31,59 @@ export const BlocksTable = styled(Table)`
   }
 `;
 
-export const Abbr = styled.abbr`
-  color: ${props => props.theme["--text-color-normal"]};
-  text-decoration: none;
-`;
-
-//@todo needed?
-const StateCell = styled(Table.Td)`
-  color: ${props => props.theme["--text-color-normal"]};
-`;
-
-const Row = ({ name, state, height }) => (
+const Row = ({ name, state, height, loading }) => (
   <Table.Tr>
-    <Table.Td>
-      <Link to={"/name/" + name}>{name}</Link>
+    <Table.Td width="75%">
+      {loading ? <Skeleton /> : <Link to={"/name/" + name}>{name}</Link>}
     </Table.Td>
-    <StateCell>{state}</StateCell>
     <Table.Td>
-      <Link to={"/block/" + height}>{height}</Link>
+      {loading ? <Skeleton /> : state}
+      </Table.Td>
+    <Table.Td>
+      {loading ? <Skeleton /> : <Link to={"/block/" + height}>{height}</Link>}
```

Comment:
I'm concerned this isn't the best way to construct the skeleton...

Reply:
Fixed and simplified together